### PR TITLE
doc pass

### PR DESCRIPTION
**Pull request by tjheffner**
## Description
Relates to #[17588](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17588).

reviews & updates these readmes:
- analytics
- broken-links
- caching
- code-guidelines

renames & updates:
- api-data -> queries

moved & still need to flesh out:
- src/templates/README.md -> templates

### CODE REVIEW

**Review by ryguyk — COMMENTED** [review]
A few thoughts. Not critical, but might be good to consider.

File: READMEs/queries.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
+# Queries
```
[the remark this answers is not in the document]
Comment (tjheffner) [reply READMEs/queries.md:1]: I moved a few things around from the `How Queries works` section to this top level, and added some words around the interface provided by next-drupal-query. Let me know if that feels clearer for you

**Review by ryguyk — APPROVED** [review]
I love the updates. Again, very nicely done!

**Review by jtmst — APPROVED** [review]
These look really good, I particularly like the detail around the query docs. Nice work

**Pull request merged**